task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -170,7 +170,17 @@ class PutVariable(BaseModel):
     type: Literal["PutVariable"] = "PutVariable"
 
 
+class RTIFPayload(BaseModel):
```

Comment:
Nit: Elsewhere in this we don't use the term "Payload" so lets call this. (Payload made sense for where this was used in the API server, just not here)
```suggestion
class SetRenderedFieldsBaseModel):
```

or something like it

Reply:
Yeah Yeah. I was torn on whether to continue with the same name. `SetRenderedFieldsBaseModel` looks fine!